phenopacketlab-core/src/main/java/org/monarchinitiative/phenopacketlab/core/ValidateService.java

```diff
@@ -0,0 +1,29 @@
+package org.monarchinitiative.phenopacketlab.core;
+
+import org.phenopackets.phenopackettools.validator.core.ValidationResult;
+import org.phenopackets.phenopackettools.validator.core.ValidationResults;
+import org.phenopackets.phenopackettools.validator.core.ValidationWorkflowRunner;
+import org.phenopackets.phenopackettools.validator.jsonschema.JsonSchemaValidationWorkflowRunner;
+import org.phenopackets.schema.v2.PhenopacketOrBuilder;
+
+public class ValidateService {
```

Comment:
Should the Spring annotation be added to make this a service?

Reply:
The bean is declared here  https://github.com/TheJacksonLaboratory/PhenopacketLab/pull/179/files/32ad8106a5bd244481c4bec7779824b38d3b61d6#diff-3336ffc5eae271b11cf6b9bbe07d71c7928465623113ac92820d53eca51bd0ac